server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/CompactableUtils.java

```diff
@@ -441,10 +441,17 @@ private UserCompactionHelper(CompactionConfig compactionConfig, Tablet tablet,
       }
 
       if (selectedFiles.isEmpty()) {
-        tablet.setLastCompactionID(compactionId);
-
-        MetadataTableUtil.updateTabletCompactID(tablet.getExtent(), compactionId,
-            tablet.getTabletServer().getContext(), tablet.getTabletServer().getLock());
+        Boolean updateMetadataTabletFail = false;
```

Comment:
Should use primitive, rather than boxed form.

```suggestion
        boolean updateMetadataTabletFail = false;
```